update-stemcell.sh

```diff
@@ -11,9 +11,10 @@
 
 set -o errexit -o nounset
 
-REPO="${REPO:-scf}"
+REPO="${REPO:-../../github.com/SUSE/scf}"
```

Comment:
This default is making the assumption that configgin and scf repos are located in a `$GOPATH/src` style layout, which is somewhat questionable, given that neither of them are Go projects... It is also out-of-sync with the comment in line 8.

But whatever, it is user-overridable, so good enough for now.